### PR TITLE
Issue#289-export_cooperation_endpoints_example

### PR DESCRIPTION
**Pull request by Illia-Shchur**
dev
## ZenHub

* https://app.zenhub.com/workspaces/home-project-5f7b77ff8db73a001cb17009/issues/ita-social-projects/home/289

## Code reviewers
- [ ] @kh0ma
- [ ] @bbogdasha
- [ ] @klaeck
- [ ] @Kizimov
- [ ] @IhorSamoshost
- [ ] @Vladyslav-Frolov

## Summary of issue

provide backend open-api design for export cooperation functionality.

## Summary of change

Added export endpoints and all necessary payloads.

## Testing approach

no testing.

## CHECK LIST
- [x]  СI passed
- [x]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [x]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: home-open-api/src/main/resources/yaml/openapi.yaml [hunk 1]
```diff
@@ -498,7 +498,118 @@ paths:
           $ref: '#/components/responses/NotFound'
         default:
           $ref: '#/components/responses/InternalServerError'
-
+  /cooperation/{id}/tasks:
+    post:
```
Comment (IhorSamoshost) [inline home-open-api/src/main/resources/yaml/openapi.yaml:502]: In Issue#292 for these method and endpoint other requestBody and success responce are provided. It is need to decide which of these options is correct.

File: home-open-api/src/main/resources/yaml/openapi.yaml [hunk 2]
```diff
@@ -1791,6 +1902,74 @@ components:
...
+          format: int64
+          example: 1
+        ownerships:
+          type: integer
+          format: int64
+          example: 1
+    ReadXlsxFile:
+      type: string
+      format: binary
+    CreateTask:
```
Comment (IhorSamoshost) [inline home-open-api/src/main/resources/yaml/openapi.yaml:1960]: In Issue#292 this schema has other structure though it is not used there. Which is right? I think issues 289 and 292 should be garmonized.

File: home-open-api/src/main/resources/yaml/openapi.yaml [hunk 3]
```diff
@@ -1791,6 +1902,74 @@ components:
           maxLength: 13
           type: string
           example: "+380966617305"
+    CooperationDataScope:
+      type: string
+      enum:
+        - ALL
+        - ADRESS
```
Comment (IhorSamoshost) [inline home-open-api/src/main/resources/yaml/openapi.yaml:1909]: suggestion: "ADDRESSES"

File: home-open-api/src/main/resources/yaml/openapi.yaml [hunk 4]
```diff
@@ -1791,6 +1902,74 @@ components:
...
+        - ERROR
+        - PROCESSING
+        - ACCEPTED
+        - DECLINED
+    TaskType:
+      type: string
+      enum:
+        - IMPORT
+        - EXPORT
+    ReadTask:
```
Comment (IhorSamoshost) [inline home-open-api/src/main/resources/yaml/openapi.yaml:1925]: In Issue#292 property "data_scope" is absent. Maybe there should be perent model "ReadTask" which extends by ReadImportTask snd ReadExportTask?

File: home-open-api/src/main/resources/yaml/openapi.yaml [hunk 5]
```diff
@@ -1791,6 +1902,74 @@ components:
...
+          type: integer
+          format: int64
+          example: 1
+        type:
+          $ref: '#/components/schemas/TaskType'
+        data_scope:
+          $ref: '#/components/schemas/CooperationDataScope'
+        status:
+          $ref: '#/components/schemas/TaskStatus'
+    ReadReport:
```
Comment (IhorSamoshost) [inline home-open-api/src/main/resources/yaml/openapi.yaml:1938]: This schema in issue#292 is different. So, I think it must be unified or made two models via inheritance.

**Comment by kh0ma** [comment]
It was closed prior to https://github.com/ita-social-projects/Home/pull/323

**Pull request closed without merging**